### DISCUSSION TITLE
Svelte example using revogr-clipboard

### DISCUSSION THREAD

**Discussion by doukai** [revolist/revogrid#605] [General]
**Check if issue exists**
I searched the documentation for examples of using revogr-clipboard with svelte, but to no avail. Could you give me a simple example of how to use it? I would be very grateful.

**Describe the issue**
Svelte example using revogr-clipboard

**Comment by revolist** [comment]
Hello @doukai,

Thank you for reaching out! 😊

The revogr-clipboard functionality is designed to work out of the box, enabling basic copy and paste operations without additional configuration. If you’re looking for a simple example, you should be able to use the clipboard features directly in your Svelte components. [Check it out.](https://stackblitz.com/edit/vitejs-vite-dvilmw)

However, if you’re interested in more advanced use cases or specific scenarios, could you please clarify what exactly you’re looking to achieve? Providing details about the cases you want to cover will help us assist you better.

Looking forward to your response!

Best,
Revolist Team
